__tests__/config/helpers/strategyResolver.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+import accountResolver from './accountResolver';
+import setupVerifier from './setupVerifier';
+
+import { GS_DEFAULT_TEST_PROVIDER_URL } from '../constants';
+import { setIfNullish } from './env';
+import { BaseUrl } from '../../../src/constants';
+
+const LOCAL_DEVNET_NOT_RUNNING_MESSAGE = `
+Local devnet is not running. In order to properly run it you need to do the following: \n
+  - Go to the: https://hub.docker.com/r/shardlabs/starknet-devnet-rs/tags
+  - Find the latest tag and copy the "docker pull" command
+  - Run Docker on your machine
+  - Run the command: "docker pull shardlabs/starknet-devnet-rs:latest"
+`;
+
+class StrategyResolver {
+  private isRsDevnet = false;
+
+  private isRpc = false;
```

Comment:
@tabaktoni Should we rename this to be `isRpcNode`, maybe it's more specific if I understood this correctly?

Reply:
I think we can rename to isRpcNode